onnxscript/function_libs/torch_lib/graph_building.py

```diff
@@ -699,15 +715,36 @@ def to_model_proto(
             keep_initializers_as_inputs=False,
             custom_opsets={},
             add_node_names=True,
-            # TODO(#493): Passing in this instead of reading from env.
-            # User must put the exported model file in the same folder to launch ORT.
-            onnx_file_path=os.path.join(
-                os.getenv("EXTERNAL_ONNX_INITIALIZER_FOLDER", ""), "dummy_model_path.onnx"
-            ),
             node_attr_to_name={},
         )
 
-        onnx_model = onnx.load_from_string(proto)
+        cache_model_to_disk = include_initializers and large_model
+
+        if cache_model_to_disk:
```

Comment:
Whether or not storing initializers should be controlled by a user flag. Assume that I export a 1GB model on remote machine. I want to visualize it locally. I really don't want to download its initializers with home internet. If this flag can be turned on, I will be able to just download the structure of model and debug faster.

Reply:
I think once the user get the model proto, they can do whatever they want? A user has full control when they get the dynamo export output as an object. 

Further yet include_initializers is already an argument

Reply:
Agree with @wschin 's goal and @justinchuby 's explanation. A thing to consider for `ExportOutput.save` or `ExportOutputSerializer`.